web-new/src/components/player/LivePlayer.tsx

```diff
@@ -0,0 +1,175 @@
+import WebRtcPlayer from "./WebRTCPlayer";
+import { CameraConfig } from "@/types/frigateConfig";
+import AutoUpdatingCameraImage from "../camera/AutoUpdatingCameraImage";
+import ActivityIndicator from "../ui/activity-indicator";
+import { Button } from "../ui/button";
+import { LuSettings } from "react-icons/lu";
+import { useCallback, useMemo, useState } from "react";
+import { Card, CardContent, CardHeader, CardTitle } from "../ui/card";
+import { Switch } from "../ui/switch";
+import { Label } from "../ui/label";
+import { usePersistence } from "@/context/use-persistence";
```

Comment:
This context is missing from this PR and doesn't exist on the 0.14 branch.

Reply:
Looks like it was moved to `hooks` and missed here.

Reply:
Yeah missed that one, fixed in https://github.com/blakeblackshear/frigate/pull/8981